Example/Podfile

```diff
@@ -15,7 +15,7 @@ target 'Emission' do
   pod 'Emission', :path => '../'
 
   # As this runs dev, we need the developer web socket
-  pod 'React', :path => react_path, :subspecs => %w(RCTWebSocket)
+  pod 'React', :path => react_path, :subspecs => %w(DevSupport BatchedBridge RCTAnimation)
```

Comment:
`ArtistCard` uses the pure JS animation library, so it’s not using the pod like the `Gene` one is.